user-management/src/app/shared/users-list/userslist.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, OnInit, OnChanges, DoCheck } from '@angular/core';
```

Comment:
remove unused imports OnChanges and You havent written anything in DoCheck remove them if not used